### PR TITLE
[DRAFT] Contacts Pages

### PR DESCRIPTION
**Pull request by jperson1**
Closes https://github.com/GSA-TTS/FAC/issues/4442.

See https://github.com/GSA-TTS/FAC/issues/4231 for more context on implementing agency contact pages.

### CODE REVIEW

File: src/contact-resources/cognizant-agency-contacts.njk [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          <th scope="row">{{line.Agency}}</th>
+          <th>{{line["Cognizant Agency POC Name"]}}</th>
+          <th>{{line["Cognizant Agency POC Email"]}}</th>
+        </tr>
+      {% endfor %}
+    </tbody>
+  </table>
+</div>
+
+<p>PDF LINK GOES HEREISH!?!?</p>
```
Comment (danswick) [inline src/contact-resources/cognizant-agency-contacts.njk:36]:
```suggestion
```
No PDF link needed on this page.

Comment (jperson1) [reply src/contact-resources/cognizant-agency-contacts.njk]: Oops. Thanks!

File: src/_includes/resources_page.njk [hunk 2]
```diff
@@ -24,19 +24,30 @@ layout: layout.njk
                         </p>
                     </div>
                     <div class="usa-card__footer padding-top-4">
-                        {% if "https" in item.data.link %}
-                            <a class="usa-button" href="{{item.data.link}}">View</a>
-                        {% else %}
-                            <a class="usa-button" href="{{ config.baseUrl}}{{item.data.link}}">View</a>
-                        {% endif %}
+                        {# If the reference link is local, go right to it with the proper baseUrl. If not, open in a new tab. #}
```
Comment (danswick) [inline src/_includes/resources_page.njk:27]: Clever!

**Review by danswick — APPROVED** [review]
Looks great. Thanks, James!

**Pull request merged**